test/module/irohad/validation/query_execution.cpp

```diff
@@ -696,13 +803,14 @@ TEST_F(GetAccountTransactionsTest, MyAccountValidCase) {
 
   EXPECT_CALL(*block_query, getAccountTransactions(admin_id))
       .WillOnce(Return(txs_observable));
-  auto response = validateAndExecute();
-  auto cast_resp = std::static_pointer_cast<TransactionsResponse>(response);
 
-  auto TxWrapper = make_test_subscriber<CallExact>(txs_observable, N);
-  TxWrapper.subscribe(
-      [this](auto val) { EXPECT_EQ(admin_id, val->creatorAccountId()); });
-  ASSERT_TRUE(TxWrapper.validate());
+  auto response = validateAndExecute(query);
+  auto cast_resp = boost::get<w<shared_model::interface::TransactionsResponse>>(
+      response->get());
+
+  for (const auto &tx : cast_resp->transactions()) {
+    EXPECT_EQ(admin_id, tx->creatorAccountId());
```

Comment:
Looks like suspicious changes: the previous version change the internal invariant and number of calls. This version checks only invariant.

Reply:
Now we don't have observable, in response we have only raw vectors

Reply:
Well, this means that you miss the number of calls (same as a number of elements) invariant.